iaac/terraform/kubernetes-addons/amazon-controller-for-k8s-sagemaker/sagemaker_studio_access.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
this will be added to repo soon, we should remove it

Reply:
it got added https://github.com/awslabs/kubeflow-manifests/blob/main/awsconfigs/infra_configs/iam_ack_oidc_sm_studio_policy.json

Reply:
changed

Reply:
done